tests/test-cluster.py

```diff
@@ -83,6 +86,28 @@ def _setup_lxc(self, channel_or_snap):
                 ).split()
             )
             time.sleep(20)
+
+            # Set launch configurations before installing microk8s
+            if launch_configs is not None:
+                print("Setting launch configurations")
+                cmd_prefix = "/snap/bin/lxc exec {}  -- script -e -c".format(self.vm_name).split()
+                cmd = ["mkdir -p /root/snap/microk8s/common/"]
+                subprocess.check_output(cmd_prefix + cmd)
+
+                file_path = '/microk8s.yaml'
```

Comment:
Are we sure we are writing the right path to the right place? https://microk8s.io/docs/add-launch-config